src/select/select-component.js

```diff
@@ -656,7 +656,7 @@ class Select extends React.Component<PropsT, SelectStateT> {
             key={`value-${i}-${value[this.props.valueKey]}`}
             removeValue={() => this.removeValue(value)}
             disabled={disabled}
-            overrides={{MultiValue: overrides.MultiValue}}
+            overrides={{Tag: overrides.Tag}}
```

Comment:
Is this a regression? Will it need to union the two values like `overrides.MultiValue || overrides.Tag`?

Reply:
oh yeah, good catch, thanks Chase!